app/server/tests/api-login.test.js

```diff
@@ -0,0 +1,49 @@
+const { AuthFunctions } = require('./functions/authFunctions')
+
+let users = new AuthFunctions(); // build class for user actions
+
+describe('Testing optimal inputs for login', () => {
+    let response;
+
+    beforeAll(async () => {
+        //register user if not already done
+        await users.register('test', 'test@gov.bc.ca', 'Test123!');
+        response = await users.login('test', 'Test123!');
+    });
+
+    test('Test account can log in - returns 201', () => {
+        expect(response.status).toBe(201);
+    });
+
+    test('Token and refresh token are received upon login', () => {
+        let token = response.body.token;
+        let refreshToken = response.body.refreshToken;
+
+        expect(token).toBeTruthy();
+        expect(typeof token).toBe('string');
+        expect(refreshToken).toBeTruthy();
+        expect(typeof refreshToken).toBe('string');
+    });
+});
+
+describe('Testing sub-optimal login input', () => {
```

Comment:
what is sub optimal?

Reply:
Was thinking inputs that the ideal user doesn't put in. Could change to something like correct vs incorrect inputs. Thoughts?

Reply:
what is the definition for correct vs incorrect?

Reply:
Whether or not the response is expected to be successful.

Reply:
@akroon3r  Updated descriptions. Is that clearer?